test/types/schema.test.ts

```diff
@@ -388,9 +388,9 @@ export function autoTypedSchema() {
     array3?: any[];
```

Comment:
Mark these as required paths if you liked the suggested changes in https://github.com/Automattic/mongoose/pull/12007/files#r910903143 :
```TS
    array1: string[];
    array2: any[];
    array3: any[];
    array4: any[];
    array5: any[];
    ```